packages/expo-av/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 ### 🛠 Breaking changes
 
+- Replace `RecordingOptions` related constants with enums. ([#17954](https://github.com/expo/expo/pull/17954) by [@Simek](https://github.com/Simek))
+  - Replace `Audio.RECORDING_OPTION_ANDROID_AUDIO_ENCODER_*` with `Audio.AndroidAudioEncoder.*`.
```

Comment:
Nested lists or other markdown elements except text and links will not work correctly with some of our scripts. Please read [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md#changelog-entries) guide.
It would be better to just include those changes in PRs description and refer there for more details — that's why we put the link to the PR, people can find more details in there.

Reply:
Yeah, I remember about that, but I afraid to not list the breaking changes migration steps. 

When I add general information about breaking change and migration details are only in PR it often results in few issues popping out in repository, when people update their packages.

Also, the new docs will be only available in `unversioned` docs, which means people will see the old documentation on PROD until next SDK release, which make finding the solution for problem after upgrade is even harder.